runtime/src/snapshot_utils.rs

```diff
@@ -536,6 +677,112 @@ where
     Ok(ret)
 }
 
+fn deserialize_snapshot_data_files_capped<F, T>(
```

Comment:
Nice, can the previous version `deserialize_snapshot_data_file_capped` be deprecated/removed now in favor of this version?

Reply:
Yes, that can be done. I was wondering about that myself. This function is also called when deserializing the status cache[1], and that's just a single file, so I left this function here for that. I could change the status cache code to call this version, but I thought that looked clunky and didn't communicate intent well. Whatcha think?

[1] https://github.com/brooksprumo/solana/blob/4f45db66c833ab1828c668c84de05b55e26a83e8/runtime/src/snapshot_utils.rs#L1634

Reply:
Oh, sorry, I think I misunderstood ^^^

Yes, the `_capped()` versions can be collapsed into just one. That's the intent, yes?

Reply:
> Yes, the _capped() versions can be collapsed into just one. That's the intent, yes?

yeah it seems like one function should be enough to handle either just a full snapshot or a full + incremental snapshot

Reply:
Gotcha. Done!